src/views/components-view.js

```diff
@@ -26,40 +26,36 @@ export class ComponentsView extends React.Component {
             <React.Fragment>
                 <Header />
                 <main>
-                    <h2>Inserting an elment from Contentful taking the content Id as the prop.</h2>
-                    <h3>This example renders a page, consisting of other components</h3>
-
-                    <TipserElement id="2OeW1Ab8lKNcUq6dVUh79" />
-
-                    <h2>Inserting a product using the productId as prop</h2>
+                    {/* Inserting a product using the productId as prop */}
+                    <h2>Product component</h2>
                     <br />
                     <h3>Full width product view</h3>
-                    <Product productId="5ba2334a781baa0001ccdffc" />
+                    <Product productId="5c8c3cb257b6a30001357db7" />
 
                     <div style={styleMedium}>
                         <h3>Container width 980px</h3>
-                        <Product productId="5aafd24f9d25801d54d41132" />
+                        <Product productId="5c8c3cb157b6a30001357da9" />
                     </div>
                     <div style={styleSmall}>
                         <h3>Container width 767px</h3>
-                        <Product productId="5aafd24f9d25801d54d41132" />
+                        <Product productId="5c8c3cb157b6a30001357da9" />
                     </div>
                     <div style={styleXS}>
                         <h3>Container width 400px</h3>
-                        <Product productId="5aafd24f9d25801d54d41132" />
+                        <Product productId="5c8c3cb157b6a30001357da9" />
                     </div>
 
-                    <h2>Inserting an elment from Contentful taking the content Id as the prop.</h2>
-                    <h3>This example renders a page, consisting of other components</h3>
+                    {/* Inserting an elment from Contentful taking the content Id as the prop */}
+                    {/*This example renders a page consisting of other components*/}
```

Comment:
Can we remove it completely instead of commenting?

Reply:
ok. Wasn't sure